browser/src/slideshow/PresenterConsole.js

```diff
@@ -504,6 +502,9 @@ class PresenterConsole {
 
 	_onShowNotes() {
 		let elem = this._proxyPresenter.document.querySelector('#notes');
+		// Apply the 'selected' styles on show notes to display toggle effect on button
+		elem.style.filter = 'invert(1)';
+		elem.style.backgroundColor = 'black';
```

Comment:
can we put that into method? and reuse in the second place - might be useful for other buttons later
`toggleButtonState(on: boolean)`